app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+import { th } from '@pubsweet/ui'
+import Select from 'react-select'
+
+const Root = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: ${th('gridUnit')};
+`
+
+const SelectLimitMessage = styled.p`
+  color: ${th('colorSuccess')};
+`
+
+const eLifeOptions = [
+  {
+    value: 'biochemistry-chemical-biology',
+    label: 'Biochemistry and Chemical Biology',
+  },
+  { value: 'cancer-biology', label: 'Cancer Biology' },
+  { value: 'cell-biology', label: 'Cell Biology' },
+  {
+    value: 'chromosomes-gene-expression',
+    label: 'Chromosomes and Gene Expression',
+  },
+  {
+    value: 'computational-systems-biology',
+    label: 'Computational and Systems Biology',
+  },
+  {
+    value: 'developmental-biology-stem-cells',
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
+    )
+
+    this.state = {
+      selectedOptions: savedOptions,
+      hasReachedMultiselectLimit: false,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+
+    this.customReactSelectStyles = {
+      multiValue: (base, state) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorPrimary,
+      }),
+    }
+  }
+
+  // https://github.com/JedWatson/react-select/blob/v2/src/Select.js
+  // react-select exposes onChange: (ValueType, ActionMeta) => void
+  handleChange = selectedOptions => {
+    if (selectedOptions.length <= 2) {
+      this.setState({ selectedOptions, hasReachedMultiselectLimit: false })
+      if (this.props.onChange) this.props.onChange(selectedOptions)
+    }
+    if (selectedOptions.length >= 2) {
+      this.setState({ hasReachedMultiselectLimit: true })
+    }
+  }
+
+  handleBlur = e => {
+    if (this.props.onBlur) this.props.onBlur(e)
```

Comment:
same: required anyway

Reply:
but maybe it shouldn't be required?

Reply:
Removing `if (this.props.onBlur)` makes sense to me. But why would I not make it required?

When a user leaves the `react-select` component, this will cause a blur event. If my code just says
```
handleBlur = e => this.props.onBlur(e)
```
but onBlur is not a required prop, then won't this cause an error?

Reply:
yes if you make it not required you would need to keep the if!